api/environments/dynamodb/wrappers/base.py

```diff
@@ -43,3 +65,50 @@ def query_get_all_items(self, **kwargs: dict) -> typing.Generator[dict, None, No
                 break
 
             kwargs["ExclusiveStartKey"] = last_evaluated_key
+
+    def batch_write(self, items: list) -> None:
+        with self.table.batch_writer() as batch:
+            for item in items:
+                batch.put_item(Item=item)
+
+
+class DAXWrapper(BaseDynamoWrapper):
+    def __init__(self) -> None:
+        super().__init__()
+        self._dax_table: typing.Optional["Table"] = None
+
+    @property
+    def dax_table(self):
+        if not self._dax_table and settings.DAX_ENDPOINT:
+            self._dax_table = AmazonDaxClient.resource(
+                endpoint_url=settings.DAX_ENDPOINT, config=Config(tcp_keepalive=True)
+            ).Table(self.table_name)
+
+        return self._dax_table
+
+    def get_item(
+        self, key: Mapping[str, "TableAttributeValueTypeDef"]
+    ) -> "GetItemOutputTableTypeDef":
+        try:
+            return self.dax_table.get_item(Key=key)
+        except Exception as e:
+            logger.error("Error getting item from DAX: %s", e)
+            return super().get_item(key)
+
+    def delete_item(
+        self, key: Mapping[str, "TableAttributeValueTypeDef"]
+    ) -> "DeleteItemOutputTableTypeDef":
+        try:
+            return self.dax_table.delete_item(Key=key)
+        except Exception as e:
```

Comment:
Very open to any ideas here to handle this better (Catching everything here because amazondax does not have a list of exceptions that it will raise)